tgui/packages/tgui/interfaces/Achievements.jsx

```diff
@@ -1,16 +1,13 @@
 import { useState } from 'react';
 
-import { useBackend, useLocalState } from '../backend';
+import { useBackend } from '../backend';
 import { Box, Flex, Icon, Table, Tabs, Tooltip } from '../components';
 import { Window } from '../layouts';
 
 export const Achievements = (props) => {
   const { data } = useBackend();
   const { categories } = data;
```

Comment:
not your fault but i am paranoid
```suggestion
  const { categories = [] } = data;
```